packages/adapter/src/types.ts

```diff
@@ -0,0 +1,8 @@
+import { ChainAdapter, TaskScheduler } from "./chains";
+
+export interface SendExtrinsicResult {
+  events: any[];
+  blockHash: any;
+}
```

Comment:
Add a json for accountType.

AccountType: {
"accountId20": "accountId20"
}

Reply:
Fixed